docs/proxy/google_serp_proxy/nodejs_examples.md

```diff
@@ -132,23 +122,22 @@ Get a list of results from the US for keyword `wikipedia` and parse them through
                 if (request.resourceType() !== 'document') return request.abort();
                 return request.continue();
             });
-            await page.goto(url, { waitUntil: 'domcontentloaded' });
+            await page.goto(url, {waitUntil: 'domcontentloaded'});
```

Comment:
This is wrongly linted, our ESLint keeps the spaces.

On the other hand, I think we can remove this `waitUntil` completely. It's only confusing and has nothing to do with proxy.

Reply:
Ok, removed the `waitUntil` opt